commit 94d211c87955d79fe6a74d7385bff014cfc95192
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Mon May 30 19:03:01 2016 +0200

    Add more hurd exception to local headers list
    
    	* scripts/check-local-headers.sh (exclude): Add hurd/ihash.h, and
    	include .*-.*/ in addition to .*-.*-.*/ (i.e. i386-gnu in addition to
    	i386-linux-gnu).

diff --git a/scripts/check-local-headers.sh b/scripts/check-local-headers.sh
index 0670da1..4cae4db 100755
--- a/scripts/check-local-headers.sh
+++ b/scripts/check-local-headers.sh
@@ -33,7 +33,7 @@ exec ${AWK} -v includedir="$includedir" '
 BEGIN {
   status = 0
   exclude = "^" includedir \
-    "/(.*-.*-.*/|)(asm[-/]|arch|linux/|selinux/|mach/|device/|hurd/(((hurd|ioctl)_types|paths)\\.h|ioctls\\.defs)|cthreads\\.h|gd|nss3/|c\\+\\+/|sys/(capability|sdt(|-config))\\.h|libaudit\\.h)"
+    "/(.*-.*-.*/|.*-.*/|)(asm[-/]|arch|linux/|selinux/|mach/|device/|hurd/(((hurd|ioctl)_types|paths)\\.h|ioctls\\.defs|ihash\\.h)|cthreads\\.h|gd|nss3/|c\\+\\+/|sys/(capability|sdt(|-config))\\.h|libaudit\\.h)"
 }
 /^[^ ]/ && $1 ~ /.*:/ { obj = $1 }
 {
